sentry_sdk/tracing.py

```diff
@@ -1311,16 +1315,6 @@ def containing_transaction(self):
         )
         return self.root_span
 
-    @containing_transaction.setter
-    def containing_transaction(self, value):
```

Comment:
Please add to migration guide -- I don't expect anyone to actually be setting this outside of the sdk but we should have it documented